Common/Constants.cs

```diff
@@ -136,5 +136,23 @@ public static TimeSpan MaxAllowedExecutionTime
         public const string PackageNameTxt = "packagename.txt";
         public const string AppOfflineFileName = "app_offline.htm";
         public const string AppOfflineKuduContent = "Created by kudu";
+
+        public const string OneDeploy = "OneDeploy";
+
+        //OneDeploy Deploy Types
+        public const string War = "war";
+        public const string Jar = "jar";
+        public const string Lib = "lib";
+        public const string Startup = "startup";
+        public const string Ear = "ear";
+        public const string Static = "static";
+        public const string Zip = "zip";
+
+        //OneDeploy Webapp Frameworks 
+        public const string Framework = "WEBSITE_FRAMEWORK";
+        public const string Tomcat = "tomcat";
+        public const string JavaSE = "javase";
+        public const string JavaEE = "javaee";
+
```

Comment:
tomcat, javase and javaee are not exactly equivalent concepts, but we can revisit this later.